jenkins-x-lint.yml

```diff
@@ -14,12 +14,16 @@ pipelineConfig:
             sh: "./add-pr-build-comment"
         - name: lint-checks
           parallel:
```

Comment:
Should this be changed to `steps` instead? Otherwise it will try to run the `pip install` in parallel to the actual checks.